src/pot.app.src

```diff
@@ -26,7 +26,10 @@
   {env,[]},
   {modules, []},
 
-  {maintainers, ["Yuce Tekol"]},
+  {maintainers, [
```

Comment:
I think we should remove this, the field is deprecated in Hex and results in a publish error on the latest versions of rebar3: https://github.com/tsloughter/rebar3_hex/issues/125